packages/admin/src/form/RangeSlider.tsx

```diff
@@ -0,0 +1,102 @@
+import { FormControl, Input, Slider } from "@material-ui/core";
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import * as sc from "./RangeSlider.sc";
+interface IRangeSliderProps extends FieldRenderProps<any, HTMLInputElement> {
+    name: string;
+    min: number;
+    max: number;
+    startAdornment?: string | React.ReactElement;
+    endAdornment?: string | React.ReactElement;
+    thumb?: React.ElementType<React.HTMLAttributes<HTMLSpanElement | HTMLDivElement>>;
+    components?: {
+        inputFieldContainer?: React.ComponentType;
+    };
+}
+export const RangeSlider: React.FunctionComponent<IRangeSliderProps> = ({
+    min,
+    max,
+    thumb,
+    components,
+    startAdornment,
+    endAdornment,
+    input: { name, onChange, value: fieldValue },
+}) => {
+    const [minInput, setMinInput] = React.useState(fieldValue.min || 0);
+    const [maxInput, setMaxInput] = React.useState(fieldValue.max || 0);
+    const InputFieldContainer = components && components.inputFieldContainer ? components.inputFieldContainer : sc.InputFieldContainer;
+    const handleSliderChange = (event: React.ChangeEvent<HTMLInputElement>, newValue: number[]) => {
+        setMinInput(newValue[0]);
+        setMaxInput(newValue[1]);
+        onChange({ min: newValue[0], max: newValue[1] });
+    };
+
+    return (
+        <sc.Wrapper>
+            <sc.InputsWrapper>
+                <InputFieldContainer>
+                    <FormControl>
+                        <Input
```

Comment:
Damit die Input's gleich ausschauen, wie die restlichen im Comet-Admin, verwende bitte `InputBase` mit `getDefaultVPAdminInputStyles`, statt den standard Mui-Inputs. 

```suggestion
                        <InputBase
                            style={getDefaultVPAdminInputStyles(theme)}
```